inc/class-formatter.php

```diff
@@ -42,21 +42,39 @@ public function __construct( &$assoc_args, $fields = null, $prefix = false ) {
 	 *
 	 * @param array $items
 	 */
-	public function display_items( $items, $include_total = true ) {
+	public function display_items( $order, $orderby, $items, $include_total = true ) {
```

Comment:
Changing this argument order is a backwards compatibility break (however unlikely it may be). Can we add the arguments to the end?